spec/user_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Tom') }
+  let(:user) { subject }
+
+  it 'Name must not be blank.' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'PostsCounter must be an integer greater than or equal to zero.' do
+    subject.postsCounter = -1
```

Comment:
Nice job so far👍🏼. As a requirement, all tests must be passing😉. currently, one of your tests is still failing which indicates a no method error. this is because you used the wrong name for the post counter😉. kindly do well to fix this😉.

```suggestion
    subject.posts_counter = -1
```


<img width="854" alt="image" src="https://github.com/siddghosh108/blog-app/assets/64643995/12bfdbf8-e208-41df-b1f2-c2da80ab5aaf">

Reply:
Made the exact change asked for in the user_spec.rb

it 'posts_counter must be an integer greater than or equal to zero.' do
subject.posts_counter = -1
expect(subject).to_not be_valid
end

Reply:
Nice job, Please ensure your tests are passing before requesting another review😉.

Reply:
Yes i tested with Rspec spec and got the following result
macbookair@Macs-MacBook-Air blog-app % rspec spec                                                                              
............

Finished in 1.06 seconds (files took 9.95 seconds to load)
12 examples, 0 failures